qc_filtering/annotation_functions.R

```diff
@@ -0,0 +1,143 @@
+
+# load in annotation files 
+# (all orginally sources from www.pseudomonas.com)
+
+PAO1_orth <- read.csv('~/Dropbox (Hogan Lab)/Resources/Annotations/Pseudomonas_aeruginosa_PAO1_107_orthologs.csv', stringsAsFactors = FALSE)
```

Comment:
Not sure if someone will want to rerun your QC steps, but if that's the case you'll probably want to update the paths to where users can access it

Reply:
For sure, good catch.